[BUG] `npm ci` lists `npm install` flags that don't work for `npm ci`

### Is there an existing issue for this?

- [X] I have searched the existing issues

### This issue exists in the latest npm version

- [X] I am using the latest npm

### Current Behavior

```bash
$ npm ci --global
npm ERR! code ECIGLOBAL
npm ERR! `npm ci` does not work for global packages
```

### Expected Behavior

[The documentation](https://docs.npmjs.com/cli/v9/commands/npm-ci#global) says that the global flag is allowed:
> Operates in "global" mode, so that packages are installed into the prefix folder instead of the current working directory. See [folders](https://docs.npmjs.com/cli/v9/configuring-npm/folders) for more on the differences in behavior.
> 
> * packages are installed into the `{prefix}/lib/node_modules` folder, instead of the current working directory.
> * bin files are linked to `{prefix}/bin`
> * man pages are linked to `{prefix}/share/man`

### Steps To Reproduce

1. execute `npm ci -g`.

### Environment

The code raising the error is here:
https://github.com/npm/cli/blob/1c3612cc2fe9974166b5aab92477b289a768c1e7/lib/commands/ci.js#L18-L22

and there are tests for this:
https://github.com/npm/cli/blob/1c3612cc2fe9974166b5aab92477b289a768c1e7/test/lib/commands/ci.js#L184-L189

so I guess my environment doesn't matter here.

--- a/lib/commands/ci.js
+++ b/lib/commands/ci.js
@@ -6,13 +6,27 @@ const log = require('../utils/log-shim.js')
 const validateLockfile = require('../utils/validate-lockfile.js')
 
 const ArboristWorkspaceCmd = require('../arborist-cmd.js')
-const Install = require('./install.js')
 
 class CI extends ArboristWorkspaceCmd {
   static description = 'Clean install a project'
   static name = 'ci'
 
-  static params = Install.params
+  // These are in the order they will show up in when running "-h"
+  static params = [
+    'install-strategy',
+    'legacy-bundling',
+    'global-style',
+    'omit',
+    'strict-peer-deps',
+    'package-lock',
+    'foreground-scripts',
+    'ignore-scripts',
+    'audit',
+    'bin-links',
+    'fund',
+    'dry-run',
+    ...super.params,
+  ]
 
   async exec () {
     if (this.npm.global) {

--- a/lib/commands/install.js
+++ b/lib/commands/install.js
@@ -16,6 +16,7 @@ class Install extends ArboristWorkspaceCmd {
   static name = 'install'
 
   // These are in the order they will show up in when running "-h"
+  // If adding to this list, consider adding also to ci.js
   static params = [
     'save',
     'save-exact',

--- a/smoke-tests/tap-snapshots/test/index.js.test.cjs
+++ b/smoke-tests/tap-snapshots/test/index.js.test.cjs
@@ -55,8 +55,6 @@ npm ERR! Usage:
 npm ERR! npm ci
 npm ERR! 
 npm ERR! Options:
-npm ERR! [-S|--save|--no-save|--save-prod|--save-dev|--save-optional|--save-peer|--save-bundle]
-npm ERR! [-E|--save-exact] [-g|--global]
 npm ERR! [--install-strategy <hoisted|nested|shallow|linked>] [--legacy-bundling]
 npm ERR! [--global-style] [--omit <dev|optional|peer> [--omit <dev|optional|peer> ...]]
 npm ERR! [--strict-peer-deps] [--no-package-lock] [--foreground-scripts]

--- a/tap-snapshots/test/lib/docs.js.test.cjs
+++ b/tap-snapshots/test/lib/docs.js.test.cjs
@@ -2210,8 +2210,6 @@ Usage:
 npm ci
 
 Options:
-[-S|--save|--no-save|--save-prod|--save-dev|--save-optional|--save-peer|--save-bundle]
-[-E|--save-exact] [-g|--global]
 [--install-strategy <hoisted|nested|shallow|linked>] [--legacy-bundling]
 [--global-style] [--omit <dev|optional|peer> [--omit <dev|optional|peer> ...]]
 [--strict-peer-deps] [--no-package-lock] [--foreground-scripts]
@@ -2229,9 +2227,6 @@ npm ci
 aliases: clean-install, ic, install-clean, isntall-clean
 \`\`\`
 
-#### \`save\`
-#### \`save-exact\`
-#### \`global\`
 #### \`install-strategy\`
 #### \`legacy-bundling\`
 #### \`global-style\`
@@ -2811,8 +2806,6 @@ Usage:
 npm install-ci-test
 
 Options:
-[-S|--save|--no-save|--save-prod|--save-dev|--save-optional|--save-peer|--save-bundle]
-[-E|--save-exact] [-g|--global]
 [--install-strategy <hoisted|nested|shallow|linked>] [--legacy-bundling]
 [--global-style] [--omit <dev|optional|peer> [--omit <dev|optional|peer> ...]]
 [--strict-peer-deps] [--no-package-lock] [--foreground-scripts]
@@ -2830,9 +2823,6 @@ npm install-ci-test
 aliases: cit, clean-install-test, sit
 \`\`\`
 
-#### \`save\`
-#### \`save-exact\`
-#### \`global\`
 #### \`install-strategy\`
 #### \`legacy-bundling\`
 #### \`global-style\`

